templates/repo/activity.tmpl

```diff
@@ -108,6 +108,12 @@
 						{{.i18n.Tr "repo.activity.git_stats_and_deletions" }}
 						<strong class="text red">{{.i18n.Tr (TrN .i18n.Lang .Activity.Code.Deletions "repo.activity.git_stats_deletion_1" "repo.activity.git_stats_deletion_n") .Activity.Code.Deletions }}</strong>.
 					</div>
+					<div class="ui attached segment">
+						<script type="text/javascript">
+						var ActivityTopAuthors = {{Json .ActivityTopAuthors | SafeJS}};
```

Comment:
Isn't JSON inherently safe JS? What will `SafeJS` do to the output? (Can it change it in a bad way somehow?).

Reply:
It just let's golang template know that it is safe js so it would not escape it